pkg/scm/git/git.go

```diff
@@ -196,8 +196,19 @@ func ParseURL(source string) (*url.URL, error) {
 }
 
 func useCopy(protoSpecified bool, source string) bool {
-	glog.V(4).Infof("useCopy proto spec %v local git repo %v has git bin %v", protoSpecified, isLocalGitRepository(source), hasGitBinary())
-	return !isLocalGitRepository(source) || !hasGitBinary()
+	glog.V(4).Infof("useCopy proto spec %v local git repo %v has git bin %v empty git repository %v", protoSpecified, isLocalGitRepository(source), hasGitBinary(), isEmptyGitRepository(source))
+
+	if isLocalGitRepository(source) {
+		if !hasGitBinary() {
+			fmt.Println("WARNING: " + source + " is a Git repository, but you do not have git installed.")
+		}
+
+		if isEmptyGitRepository(source) {
+			fmt.Println("WARNING: " + source + " is a Git repository, but is empty.")
+		}
+	}
+
+	return !isLocalGitRepository(source) || !hasGitBinary() || isEmptyGitRepository(source)
```

Comment:
I see that we're executing `isLocalGitRepository()` twice always and also executing `hasGitBinary()` and `isEmptyGitRepository()` twice sometimes. Because they're doing IO, maybe it's better to execute them only once? I mean, let's extract them to variables.